packages/apps/human-app/server/src/common/interceptors/axios-request.interceptor.ts

```diff
@@ -6,21 +6,25 @@ import { EnvironmentConfigService } from '../config/environment-config.service';
 export class AxiosRequestInterceptor {
   private readonly logger = new Logger(AxiosRequestInterceptor.name);
 
-  constructor(private configService: EnvironmentConfigService) {
+  constructor() {
     this.initializeRequestInterceptor();
   }
 
   private initializeRequestInterceptor() {
-    if (!this.configService.axiosRequestLoggingEnabled) return;
     axios.interceptors.request.use(
       (config) => {
         const { url, method, params, data, headers } = config;
-        this.logger.log('NEW REQUEST:');
-        this.logger.log(`Request URL: ${url}`);
-        this.logger.log(`Method: ${method}`);
-        this.logger.log(`Params: ${JSON.stringify(params ?? {})}`);
-        this.logger.log(`Body: ${JSON.stringify(data ?? {})}`);
-        this.logger.log(`Headers: ${JSON.stringify(headers ?? {})}`);
+        this.logger.log(
+          `
+---------------NEW REQUEST: -----------------------------------------------------------------------------
```

Comment:
Consider using structured logging, such as JSON, for better readability and parsing in systems like Graylog